utils/dist_push.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+: ${ORGDIR:="/src/github.com/TykTechnologies"}
+: ${SOURCEBINPATH:="${ORGDIR}/tyk-sync"}
+: ${DEBVERS:="ubuntu/trusty ubuntu/xenial ubuntu/bionic debian/jessie debian/stretch debian/buster"}
```

Comment:
Trusty's standard support [ended earlier](https://wiki.ubuntu.com/Releases) this year. I think we can drop it.